agent/run-unittests

```diff
@@ -28,6 +28,7 @@ function tail_and_wait {
 }
 
 trap "kill -KILL -$$ > /dev/null 2>&1" INT TERM QUIT
+export PATH=$PWD/../bin:$PATH
```

Comment:
Ditto - although I'm not sure what to use here. But I dont think you should define PATH here: we don't use getconf.py explicitly here. It is only used implicitly by the various `unittests` scripts, so if *they* set the PATH appropriately, then this script does not need to worry about it.

Reply:
resolved